guides/api-query-language-velo.md

```diff
@@ -0,0 +1,267 @@
+# API Query Language 
+
+The query language described in this article
+is implemented partially or in full by certain Wix Velo APIs that support query 
+capabilities.
+
+You may see some similarities between the Wix API Query Language
+and MongoQL. This is because the style of the Wix API Query Language
+is heavily influenced by MongoQL.
+
+## Velo query techniques
+
+2 types of query techniques are available in Velo. See your specific API to check which technique the query function supports. 
+    
+* **Without query builders**: Call query functions to retrieve a list of items using the API Query Language described in this article. Pass an object defining the query as a parameter to the query function.
+
+* **With query builders**: Call query functions that build a query to retrieve a list of items. You can recognize these query functions because they have associated `<item>QueryBuilder` and `<item>QueryResult` class objects. These query functions do not use the API query language syntax described here. 
```

Comment:
In the second bullet, I learn how to identify the technique. Recommend switching the order so that comes first.

Reply:
I originally had it in the opposite order --- great minds! 

But then I decided to reverse it because the main thrust of this article is how to work WITHOUT query builders. It seemed not right to me to make the emphasis on the query builder way. But I think you are right, I will go with my initial instinct.  We do want them to know that the query builder way is the default accepted way.